openassessment/assessment/test/test_staff.py

```diff
@@ -302,6 +302,22 @@ def test_provisionally_done(self):
         self.assertEqual(tim_workflow["score"], None)
         self.assertNotEqual(tim_workflow["status"], "done")
 
+    def test_update_with_override(self):
+        """
+        Test that, when viewing a submission with a staff override present, the workflow is not updated repeatedly.
```

Comment:
Perhaps putting the TNL ticket in the docstring will help clarify the reason for this test.